manim/_config/utils.py

```diff
@@ -605,7 +605,7 @@ def digest_args(self, args: argparse.Namespace) -> "ManimConfig":
         if args.config_file:
             self.digest_file(args.config_file)
 
-        self.input_file = Path(args.file).absolute()
+        self.input_file = Path(args.file).absolute() if args.file != "-" else args.file
```

Comment:
Perhaps a comment could be added explaining what happens if `args.file` is `"-"`, like how it takes code from stdin